events/artsy-learns-to-code-v2.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Artsy Learns to Code v2
+description: Second iteration on ALTC
+---
+
+See the original ALTC write-up [here](./artsy-learns-to-code.md) for the previous iteration's abstract and
+structure.
+
+## Abstract
+
+Let's kick off another session of Artsy Learns to Code!
+
+## Motivations for ALTC v2
+
+- It's social! Lets Artsy people interact across their normal silos
```

Comment:
Definitely – this is more and more important as we grow, and especially since sup's are not as actively attended anymore.